samples/quickstart.js

```diff
@@ -15,14 +15,73 @@
 
 'use strict';
 
-// [START asset_quickstart]
```

Comment:
Are you 100% sure this tag wasn't being used somewhere on cloud.google.com? Worried that we're getting rid of something that may be in use :)

Reply:
I think so. This is used for API doc, which will embed the code between the tags. But our API doc hasn't publish the come sample yet.